design/FY2022/enhacement_to_resilience_metric.md

```diff
@@ -0,0 +1,211 @@
+
+UDH is calculated as follows:
+
+![UDH_calc](https://github.com/NREL/EnergyPlus/blob/enhanceResilienceMetrics/design/FY2022/UDH_eq.png)
+
+where T is the indoor air temperature [°C]; t is time [h]; and x<sub>+</sub> = x if x > 0, or 0 otherwise. T<sub>threshold</sub> is the indoor cooling or heating setpoint [°C] in both the grid-on and grid-off scenarios. A similar metric, the Exceedance Degree-Hour, is recently developed by Salimi et al [4]. Instead of thresholding, this metric weights each hour by the distance from the current SET to the comfort zone [4].
+
+For reporting, four variables will be computed: heating and cooling setpoint UDH during occupied and the whole reporting period (see Table 3). For the building level, we’ll provide four summary statistics across all zones in the building: min, max, average, and sum.
+
+### 3. Discomfort-weighted Exceedance Hours ###
+
+Discomfort-weighted exceedance hours is the sum of the positive values of predicted mean vote (PMV) exceedance during occupied hours, where PMV exceedance = (PMV – threshold) for warm or very-hot conditions, and PMV exceedance = (threshold - PMV) for cool or very-cold conditions [5]. Warm, cool, very-hot, and very-cold exceedance hours are the number of hours in which occupants are uncomfortably warm (PMV>0.7), cool (PMV<−0.7), very hot (PMV>3), and very cold (PMV<−3). For example, discomfort-weighted warm exceedance hour is the sum of the positive values of (PMV - 0.7) during occupied hours, while discomfort-weighted cool exceedance hour is the sum of the positive values of (-0.7 - PMV) during occupied hours. Discomfort-weighted very-hot and very-cold exceedance hours are calculated analogously, using thresholds of 3 and -3, respectively. CBE developed an online tool to compute thermal comfort metrics including PMV, PPD, thermal sensation, SET, etc [6].
+
+For reporting, we will present the zone-level and the aggregated building-level Occupant • Hours falling into the dis-comfort PMV ranges. For the building level, we’ll provide four summary statistics across all zones in the building: min, max, average, and sum.
+
+### 4. Timespan of Report ###
+
+Resilience metrics are more often evaluated during a certain period when a building is at risk (e.g., during the power outage event or heatwave event), and the period is not necessarily the same as the whole simulation period. We propose to improve current resilience metrics tabular reporting (in the html output) by allowing users to specify the period of calculating and reporting the metrics. 
+
+We propose to add an *Output:Table:ReportPeriod* that takes an input of a Summary Report name (e.g. *ThermalResilienceSummary*). The reporting period can be defined with a start-date field and an end-date field as supplementary inputs to the *ThermalResilienceSummary* report.
+
```

Comment:
I would recommend adding a warning message if electricity is consumed at the facility during the timespan defined. This is intended to model power outages.

Reply:
Indeed, I will add this check of non-zero electricity usage during a power outage.